tests/test_object_detection.py

```diff
@@ -601,3 +604,34 @@ def test_visualize(monkeypatch, generate_single_image_file):
         },
         overlay=False,
     )
+
+
+def test_has_labels_overlap():
+    bboxes = np.array(
+        [
+            [359.0, 146.0, 472.0, 360.0],
+            [340.0, 22.0, 494.0, 323.0],
+            [472.0, 173.0, 508.0, 221.0],
+            [486.0, 183.0, 517.0, 218.0],
+            [359.0, 144.0, 470.0, 358.0],
+        ]
+    )
+    label_classes = [0, 1, 2, 3, 2]
+    is_overlaps = _has_overlap(bboxes, label_classes)
+    expected_res = np.array([True, False, False, False, True])
+    assert np.array_equal(is_overlaps, expected_res)
+
+
+@pytest.mark.parametrize("overlapping_label_check", [True, False])
+def test_swap_overlap_labels(overlapping_label_check):
+    prediction = predictions[3].copy()
+    label = labels[3].copy()
+    label["bboxes"] = np.append(label["bboxes"], [label["bboxes"][-1]], axis=0)
+    label["labels"] = np.append(label["labels"], (label["labels"][-1] + 1) % 10)
+    score = get_label_quality_scores(
+        [label], [prediction], overlapping_label_check=overlapping_label_check
+    )[0]
+    if overlapping_label_check:
```

Comment:
Write one more unit test for user facing `compute_swap_box_scores()` function to make sure expected boxes get low overlap score if overlapping_label_check() is on.

Reply:
Added here: https://github.com/cleanlab/cleanlab/blob/96b10ac0087edd5f90bf30961408a2bed7f75564/tests/test_object_detection.py#L640